tests/random_lax_test.py

```diff
@@ -1632,6 +1632,21 @@ def test_randint_out_of_range(self):
     # TODO(mattjj): enable this test if/when RngBitGenerator supports it
     raise SkipTest('8-bit types not supported with RBG PRNG')
 
+  def test_randint_uint8_bias(self):
+    # Regression test for https://github.com/jax-ml/jax/issues/27702
+    key = self.make_key(7534892)
+    n_samples = 100_000
+    n_bins = 100
+    data = jax.random.randint(key, (n_samples,), 0, n_bins, dtype='uint8')
+
+    # Check that counts within each bin are consistent with a uniform distribution
+    # (within three poisson standard deviations).
+    counts = jnp.bincount(data, length=n_bins).astype(float)
+    expected = n_samples / n_bins
+    std = np.sqrt(expected)
+    self.assertTrue(np.all(abs(counts - expected) < 3 * std),
```

Comment:
Hm, the chance of all 100 tests succeeding is only 75% (0.997**100). An alternative may be testing the mean, which would have found the original issue.

Reply:
Yeah there's always a tradeoff between sensitivity and specificity for these kinds of tests. I don't think the mean is the best option here because we're specifically trying to test against a bimodal distribution, and the mean alone is not a great measure of that.

Reply:
I switched to a KS test with a poisson distribution.